examples/notes/main.go

```diff
@@ -0,0 +1,204 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+// notes application demonstrates how to use Storj for storing notes.
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"storj.io/storj/lib/uplink"
+	"storj.io/storj/pkg/storj"
+)
+
+var (
+	satellite  = flag.String("satellite", "mars.tardigrade.io:7777", "satellite that store meta data")
+	apiKeyText = flag.String("api-key", os.Getenv("NOTES_API_KEY"), "api key for connecting to the satellite")
+	bucketName = flag.String("bucket", "notes", "bucket to store notes in")
+	password   = flag.String("password", os.Getenv("NOTES_PASSWORD"), "password for accessing data")
+)
+
+func main() {
+	flag.Parse()
+	if *apiKeyText == "" {
+		fatalf("api key is missing\n")
```

Comment:
I just wonder what's the benefit of defining `fatalf` instead of using the standard `log.Fatal` and similar functions?